background/score/activities.go

```diff
@@ -79,6 +100,12 @@ func (s *ScoreUpdateWorker) RefreshLocationStateActivity(ctx context.Context, ac
 		}
 
 		for _, profile := range profiles {
+
+			accountLocation := time.FixedZone("UTC+8", int((8 * time.Hour).Seconds()))
```

Comment:
Can you explain why we use FixedZone UTC+8 ? 
What this will be if a person in US ?

Reply:
It fixed for now since we do not have a way to get the timezone at the time a submit this PR.